ajax/libs/angular-autofields/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "angular-autofields",
+    "version": "2.1.8",
+    "description": "Automatically generate fields from a schema object and bind them to an object",
+    "filename": "autofields.min.js",
+    "keyword": [
+        "angular",
+        "bootstrap",
+        "form",
+        "fields",
+        "data-binding",
+        "generate"
+    ],
+    "license": "MIT",
+    "homepage": "https://github.com/justmaier/angular-autoFields-bootstrap",
+    "author": {
+        "name": "Justin Maier",
+        "email": "just.maier@gmail.com"
+    },
+    "npmName": "angular-autofields-bootstrap",
```

Comment:
npm ERR! 404 404 Not Found: angular-autofields-bootstrap
npm ERR! 404 
npm ERR! 404 'angular-autofields-bootstrap' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.